Mathlib/NumberTheory/Cyclotomic/Rat.lean

```diff
@@ -265,7 +267,7 @@ noncomputable def subOneIntegralPowerBasis [IsCyclotomicExtension {p ^ k} ℚ K]
       · exact Subalgebra.sub_mem _ (hζ.isIntegral (by simp)) (Subalgebra.one_mem _))
 
 @[simp]
-theorem subOneIntegralPowerBasis_gen [IsCyclotomicExtension {p ^ k} ℚ K]
+theorem subOneIntegralPowerBasis_gen [hcycl : IsCyclotomicExtension {p ^ k} ℚ K]
```

Comment:
Why do we have to name the instance here?

Reply:
It was used in a named argument a few lines below, but I've cleaned up and use a `have` statement for the local instance now.